Add Towncrier changelog fragments (#114)
ansible/pylibssh: docs/changelog-fragments/58.bugfix.rst

> Fixed sftp file get issue on py2

Reviewer: ```suggestion
Fixed sftp file get issue. On py2
```

> The file ``write()`` method returns None on py2

Reviewer: ```suggestion
the file ``write()`` method returns ``None`` on py2
```

> is bytes are written to file successfully, whereas

Reviewer: ```suggestion
if bytes are written to file successfully, whereas
```

on py3 it returns total number of bytes written

> to file. Add fix to check for the number of

Reviewer: ```suggestion
to file. Added a fix to check for the number of
```

bytes written only in the case when ``write()``

> is not returning None - by :user:`ganeshrn`

Reviewer: Things like `None` should be marked as inline-code with double backticks
```suggestion
is not returning ``None`` -- by :user:`ganeshrn`
```

> is not returning ``None`` -- by :user:`ganeshrn`

Reviewer: ```suggestion
does not return ``None`` -- by :user:`ganeshrn`
```
